collectors/channels_collector.go

```diff
@@ -178,6 +185,31 @@ func NewChannelsCollector(lnd lndclient.LightningClient, errChan chan<- error,
 		primaryNode: cfg.PrimaryNode,
 		errChan:     errChan,
 	}
+
+	// Perform an initial refresh for the cache
+	collector.refreshClosedChannelsCache()
+
+	// Start a ticker to update the cache once per 10m
+	go func() {
+		ticker := time.NewTicker(10 * time.Minute)
+		defer ticker.Stop()
+		for range ticker.C {
```

Comment:
This should also be listening on a quit channel so it can exit cleanly.

Reply:
Added this in the latest commit.

Reply:
Can you please squash and force push?